donate/templates/partials/donate-form.html

```diff
@@ -0,0 +1,145 @@
+<div class="donate-form">
```

Comment:
@nicklee I think this file needs to be removed as we're using `fragments/donate-form.html` instead now.

Reply:
I think this can be added back in now, as we have a generic "donate now" heading on both the campaign and landing pages in the latest design.

Reply:
Ahhh, we need the currency tag to be loaded in the actual campaign/landing page template it seems, so I have removed this again, as well as the header.

Reply:
I don't think this file is used at all?

Reply:
good point, moved to fragments!